packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -325,11 +325,14 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
       }),
 
       // active styles
-      ...(active &&
-        (primary
-          ? {
-              ...(iconOnly && { color: 'inherit' }),
+      ...(active && {
+        ...(iconOnly && {
+          color: v.iconOnlyColorActive,
```

Comment:
could you, please, suggest where original `inherit` value had resulted in suboptimal styles?

Reply:
In team's theme the `iconOnly` `Menu` should always have the brand color for the active items. The inherit color is black in default theme.